src/styles/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  gap: 1px;
+  justify-content: left;
+  width: 706px;
+  padding: 10px;
+  flex-wrap: wrap;
+
+  & :nth-child(n) {
```

Comment:
Why not use the `calendar__day` element here, instead of calendar's nth child?

Reply:
Initialy I develop the solution without using calendar__day class, and put it just for test porpuses, just because to test requirements and not critical for solution of the task. It's fine use this class, but I prefer not to change entire code again.